polkadot-parachains/src/primitives.rs

```diff
@@ -0,0 +1,11 @@
+/// Alias to 512-bit hash when used in the context of a transaction signature on the chain.
+pub type Signature = sp_runtime::MultiSignature;
+/// Some way of identifying an account on the chain. We intentionally make it equivalent
+/// to the public key of our transaction signing scheme.
+pub type AccountId = <<Signature as sp_runtime::traits::Verify>::Signer as sp_runtime::traits::IdentifyAccount>::AccountId;
+pub type BlockNumber = u32;
+pub type Header = sp_runtime::generic::Header<BlockNumber, sp_runtime::traits::BlakeTwo256>;
+pub type Block = sp_runtime::generic::Block<Header, sp_runtime::OpaqueExtrinsic>;
+pub type Hash = sp_core::H256;
+pub type Balance = u128;
+pub type Nonce = u32;
```

Comment:
I made https://github.com/paritytech/cumulus/pull/577 as an improvement to this file.

Reply:
Also in the future, need to add the copyright headers when adding new files, fyi